frontend/device/pocketbook/device.lua

```diff
@@ -79,9 +87,8 @@ function PocketBook:blacklistCBB()
     local C = ffi.C
 
     -- As well as on those than can't do HW inversion, as otherwise NightMode would be ineffective.
-    --- @fixme Either relax the HWInvert check, or actually enable HWInvert on PB if it's safe and it works,
-    --        as, currently, no PB device is marked as canHWInvert, so, the C BB is essentially *always* blacklisted.
-    if not self:canUseCBB() or not self:canHWInvert() then
+    -- Do this only if settings override isn't present.
+    if (not G_reader_settings:has("dev_no_c_blitter")) and (not self:canUseCBB() or not self:canHWInvert()) then
```

Comment:
The `G_reader_settings` bit is not necessary, `reader.lua` already sets the env var checked below ;).

(This is weirdly convoluted for... reasons I don't even remember :D).

Reply:
Ok, brought it back as it was.